RecoEgamma/EgammaTools/python/calibratedEgammas_cff.py

```diff
@@ -14,6 +16,11 @@
                                    )
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
 
+from Configuration.Eras.Modifier_run2_egamma_2016_cff import run2_egamma_2016
+from Configuration.Eras.Modifier_tracker_apv_vfp30_2016_cff import tracker_apv_vfp30_2016
+(run2_egamma_2016 & tracker_apv_vfp30_2016).toModify(calibratedEgammaSettings,correctionFile = _correctionFile2016ULpreVFP)
```

Comment:
this will still get applied with `run2_miniAOD_UL` and without it.
I think that an applicable requirement here is `run2_miniAOD_devel `
```suggestion
(run2_miniAOD_devel & run2_egamma_2016 & tracker_apv_vfp30_2016).toModify(calibratedEgammaSettings,correctionFile = _correctionFile2016ULpreVFP)
```

Reply:
okay then I will change this in both the 2016 UL modifiers

Reply:
@slava77 
for my own understanding the bParking will inherit all we had in the run2_miniAOD_devel or run2_miniAOD_UL

Reply:
bParking does not apply to 2016.
If this was for 2018, then we'd use `(run2_miniAOD_devel | bParking)`

Reply:
missed to include the cff file - sorry about it - done now